### ISSUE TITLE
Add information to docs on how to add additional fields in error

### ISSUE THREAD

**Issue by vladshcherbin** [jaydenseric/graphql-api-koa#8]
Jesus, I've spent so much time to find how to do it.

`http-errors` has possibility to add additional fields to error like this:

```js
createError(503, 'No work on the sabbath.', { type: 'DayError' , expose: true })
```

However, [formatError](https://github.com/jaydenseric/graphql-api-koa/blob/v4.1.1/lib/errorHandler.js#L76) function cuts all additional properties and produces only `message` field. To add this additional error fields you have to use `extensions` property like this:

```js
createError(503, 'No work on the sabbath.', { extensions: { type: 'DayError' }, expose: true })
```

This will correctly show additional error information to user:

![image](https://user-images.githubusercontent.com/6711845/79670393-3f62b880-81cb-11ea-8b3a-ecdb461b3005.png)

Please, add this to docs somewhere. I really hope it will help and save time for someone else.

❤️

**Comment by jaydenseric** [comment]
Just published [`graphql-api-koa@4.1.2`](https://github.com/jaydenseric/graphql-api-koa/releases/tag/v4.1.2), this stuff should be easier now 🙏

**Comment by vladshcherbin** [comment]
@jaydenseric thank you a lot, using it in my current project ❤️
